plugins/rapid7_insight_agent/icon_rapid7_insight_agent/util/graphql_api/api_connection.py

```diff
@@ -124,10 +121,10 @@ def quarantine_list(self, agent_hostname_list: List[str], advertisement_period:
 
         return successful_quarantine, unsuccessful_quarantine
 
-    def unquarantine_list(self, agent_hostname_list: List[str]) -> Tuple[List[str], List[str]]:
+    def unquarantine_list(self, agent_id_list: List[str]) -> Tuple[List[str], List[str]]:
```

Comment:
Can we update quarantine_list and unquarantine_list to use the existing functions quarantine and unquarantine?

Also may be possible to try/catch any other exceptions and add to failed array instead of exiting action but worth checking with ticket requestor for functionality.